sql/rename.go

```diff
@@ -209,6 +213,8 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 	if err := p.txn.Put(descKey, tableDesc); err != nil {
 		return nil, err
 	}
+	// Mark transaction as operating on the system DB.
+	p.txn.SetSystemDBTrigger()
```

Comment:
failing to set the trigger, since we have a regular gossip loop anyways, will only postpone the update (and we need to be robust in that regard anyways, because Gossip), so we should just make sure we have good tests for it and then hopefully there's no fragility.